Assets/JsonDotNet/Assemblies/AOT/Newtonsoft.Json.XML

```diff
@@ -0,0 +1,8015 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>Newtonsoft.Json</name>
```

Comment:
Is there a way to implement this with the built in Unity JsonUtilities?

Reply:
I'm 95% certain there is. I just need to do more testing.